### DISCUSSION TITLE
Github Pages issue with links

### DISCUSSION THREAD

**Discussion by YeahJenni** [community/community#40485] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do i get the links to be clickable on my site? People need to copy and paste them but i want them to be able to click on it
![image](https://user-images.githubusercontent.com/83929921/204919962-ca2b7381-7746-47bb-a5e7-f01e7a2b9dbe.png)

**Comment by radar-blip** [comment, marked as answer]
Try this and see if it works on your site. `[Audactiy Blog:](https://cdjen9.wixsite.com/audacity-announcemen)`

**Reply by YeahJenni** [reply]
OMG mate ur such a legend. TYSM!

**Reply by radar-blip** [reply]
You're welcome. I'm assuming it worked. 👍🏻

**Reply by YeahJenni** [reply]
Yessir!
